velox/docs/bindings.rst

```diff
@@ -0,0 +1,12 @@
+*****************
+Language Bindings
+*****************
+
+This includes language bindings supported by Velox. At the moment, Velox only supports Python.
```

Comment:
> Velox only supports Python.

This might be confusing as Velox also supports C++. Can we say Python and C++?

Reply:
Sure, we can. I will fix this.